src/Arcus.WebApi.Tests.Runtimes.AzureFunction.Isolated/Dockerfile

```diff
@@ -0,0 +1,26 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated6.0 AS base
+WORKDIR /home/site/wwwroot
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/runtime:3.1 as runtime3_1
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
```

Comment:
I'm wondering why we use .NET 6 as build, and 3.1 as runtime ?

Reply:
Me too 😄 . I think this is a copy of the other Docker file in the test project.

Reply:
Ah, I think this was generated by the VS feature `Add Docker support`.

Reply:
Removed it in https://github.com/arcus-azure/arcus.webapi/pull/373/commits/d5f070e69ec4715caf038dc67d187d9a080c9005